src/vnc/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.10)
 
 project(VncServer C)
 
-add_compile_options(-Wall)
+add_compile_options(-Wall -DDISABLE_SANDBOX)
```

Comment:
Please don't disable the sandbox by default.

Reply:
Ok I agree, I should add a test and disable it only for ARM 64-bits arch